internal/command/command.go

```diff
@@ -78,7 +78,7 @@ const (
 	logLevelNone  = "none"
 )
 
-func (c Command) handleUserTracking(loader flowkit.ReaderWriter) {
+func HandleUserTracking(command *cobra.Command, loader flowkit.ReaderWriter) {
```

Comment:
please move this to the bottom where helpers are. I would also rename it to: `RecordCommandUsage`

Reply:
addressed